CONTRIBUTING.md

```diff
@@ -0,0 +1,362 @@
+because of their language.
+
+## Helping in other ways
+
+Ergoplatform occasionally is able to hire developers for part time or full time positions, 
+to work on Ergo. If you are interested contact up at ergoplatform@protonmail.com.
+<!--TODO https://github.com/ergoplatform/community/issues/5, check out [the job listings](http://protocol.ai/join)-->
+
+If you'd like to help in other ways, propose your ideas on the Ergo forums at https://www.ergoforum.org.
+
+## Code Contribution Guidelines
+
+### Discuss big changes as Issues first
+
+Significant improvements should be documented as GitHub issues before anybody starts to code. 
+This gives other contributors a chance to point you in the right direction, 
+give feedback on the design, and maybe point out if related work is under way.
+
+Please take a moment to check whether an issue already exists. 
+If it does, it never hurts to add a quick "+1" or "I have this problem too". 
+This helps prioritize the most common problems and requests.
+
+### Pull Requests always welcome
+
+We are always thrilled to receive pull requests, and do our best to process them as quickly as possible. 
+Not sure if that typo is worth a pull request? Do it! We will appreciate it.
+
+If you don't get a response to a pull request within a few days, feel free to mention it in the #Ergo telegram channel. 
+We may be able to get to it faster.
+<!--TODO https://github.com/ergoplatform/community/issues/6 (Developers Channel)-->
+
+We're also trying very hard to keep Ergo focused. 
+This means that we might decide against incorporating a new feature. 
+However, there might be a way to implement that feature on top of Ergo.
+
+If your pull request is not accepted on the first try, don't be discouraged! 
+If there's a problem with the implementation, hopefully you received feedback on what to improve.
+
+### Conventions
+
+Each repository will have its own code and test conventions. 
+Please make sure to review those before jumping in. Some general conventions are listed below.
+
+#### Git
+
+We use a simple git branching model:
+
+- `master` must always work
+- create feature-branches to merge into `master`
+- all commits must pass testing so that git bisect is easy to run
+
+Just stay current with master (rebase).
+
+##### Commit messages
+
+Commit messages must start with a short subject line, followed by an optional,
+more detailed explanatory text which is separated from the summary by an empty
+line.
+
+<!--TODO https://github.com/ergoplatform/community/issues/7 -->
+<!--For some of our repositories (and we'd prefer all), we check commit messages-->
+<!--automatically using [GitCop](https://gitcop.com). This checks that commit-->
+<!--messages are properly written.-->
+
+**If you are brand new to Ergo**, you'll need to run a script that adds
+ `License` and `Signed-off-by` to each of your commits. These trailers mean that the author 
```

Comment:
I dont' think this is needed for CC0.

Reply:
We need to protect contributions from any future claims. 
Any evidence that CC0 implies the contributor bear the responsibility?
Anyway, we may require this by default, but don't have to be very strict about this.
And also sigmastate is MIT, not CC0.